cli.py

```diff
@@ -591,24 +583,24 @@ def run_ratdecoders(file_path, passed_reporter):
     for key in output:
         if key not in SUPER_LIST:
             others[key] = output[key]
-    reporter.add_metadata("other", others)
-    return {script_name: reporter.metadata}
+    report.add(metadata.Other("other", others))
```

Comment:
should this be metadata.Other(others)? or does metadata.Other accept different params then the other metadata classes?

Reply:
good catch